client/me/memberships/main.jsx

```diff
@@ -102,7 +108,11 @@ const MembershipsHistory = ( { translate, subscriptions, moment } ) => {
 		content = (
 			<CompactCard className="memberships__no-content">
 				<EmptyContent
-					title={ translate( 'No Recurring Payments found.' ) }
+					title={
+						isEnabled( 'earn/rename-payment-blocks' )
+							? translate( 'No payments found.' )
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.